package.json

```diff
@@ -108,7 +108,7 @@
     "yargs": "^14.0.0"
   },
   "dependencies": {
-    "@emotion/babel-preset-css-prop": "^10.0.27",
+    "@emotion/babel-preset-css-prop": "^11.2.0",
```

Comment:
Litt usikker på denne, vi har ikke emotion 11 som dependency. @Jonas-C prøvde å bumpe emotion men tror han kom til et punkt som lar seg ikke bumpe. Dobbeltsjekk med han for han har mer peiling på dette nå.

Reply:
Ser ikke ut som at bumping av babel-preset-css-prop påvirker de samme tingene som bumping av alt til 11. I Emotion 11 er ikke babel-preset-css-prop nødvendig uansett, da det har blitt merget inn i noe annet. MAO: Når vi får bumpet emotion kan vi fjerne det uansett.

Reply:
bumping av babel-preset-css-prop var nødvendig for å fungere med emotion/react, ser det ut til